modules/default.nix

```diff
@@ -100,29 +98,40 @@ inputs: {
   extraLuaPackages = ps: map (x: ps.${x}) vimOptions.luaPackages;
   extraPython3Packages = ps: map (x: ps.${x}) vimOptions.python3Packages;
 
+  luaConfig =
+    if vimOptions.byteCompileLua
+    then pkgs.runCommandLocal "init.lua" {text = vimOptions.builtLuaConfigRC;} "${pkgs.luajit}/bin/luajit -bd -- - $out <<< \"$text\""
+    else pkgs.writeText "init.lua" vimOptions.builtLuaConfigRC;
+
+  extraLuaFiles =
+    if vimOptions.byteCompileLua
+    then map (file: pkgs.runCommandLocal (baseNameOf file) {} "${pkgs.luajit}/bin/luajit -bd -- ${file} $out") vimOptions.extraLuaFiles
+    else vimOptions.extraLuaFiles;
+
   # Wrap the user's desired (unwrapped) Neovim package with arguments that'll be used to
   # generate a wrapped Neovim package.
   neovim-wrapped = inputs.mnw.lib.wrap pkgs {
     neovim = vimOptions.package;
     plugins = concatLists [builtStartPlugins builtOptPlugins];
     appName = "nvf";
     extraBinPath = vimOptions.extraPackages;
-    initLua = vimOptions.builtLuaConfigRC;
-    luaFiles = vimOptions.extraLuaFiles;
+    # initLua = vimOptions.builtLuaConfigRC;
+    # luaFiles = vimOptions.extraLuaFiles;
```

Comment:
Is there a reason those two haave been left out?

Reply:
Forgot to remove them